components/SwipeRow.js

```diff
@@ -141,6 +141,8 @@ class SwipeRow extends Component {
 			this.props.setScrollEnabled && this.props.setScrollEnabled(true);
 		}
 
+		this._releaseValue = this._translateX._value;
```

Comment:
Can we pass `this._translateX._value` as an argument to `manuallySwipeRow` instead of storing the value on `this`?